dependencies.gradle

```
@@ -1,11 +1,11 @@
 buildscript {
   ext.versions = [
-    'kotlin': '1.7.10',
-    'compileSdk': 32,
+    'kotlin': '1.7.21',
```

```suggestion
    'kotlin': '1.7.20',
```
Greatly simplifies using it with JetBrains Compose for which there is no .21-compatible version as of yet.

---

How can the Kotlin version used by this Gradle plugin affect JetBrains Compose compiler plugin? 🤔

---

There's only one copy on the classpath so the use of .21 will cause Gradle to use that over our declared .20 and then the Compose compiler will report that it is not compatible with this version of Kotlin.

---

Thanks, good to know.